app/views/listings/_new.html.erb

```diff
@@ -0,0 +1,67 @@
+<div id="newListingModal" role="dialog" tabindex="-1" class="modal fade text-left show" style="margin-top: 20px;">
+    <div class="modal-dialog modal-xl" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Add a new Listing</h4><button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+            </div>
+            <%= form_for(@listing) do |f|  %>
+            <div class="modal-body">
+                <div class="container">
+                    <div class="row" style="margin-top: 20px;">
+                        <div class="col-lg-4">
+                            <%= f.hidden_field :user_id, value: user_signed_in? ? current_user.id : nil %>
+                            <%= f.text_field :listing_name, required: 'required', class: "roundedFormField col-lg-12" ,placeholder: "Property Name" %>
+                        </div>
+                        <div class="col-lg-4">
+                            <%= f.select :home_type, raw("
+                            <option value='Condominium' selected>Condominium</option>
+                            <option value='Resort'>Resort</option>
+                            <option value='Hotel'>Hotel</option>
+                            <option value='Beach House'>Beach House</option>
+                            <option value='Apartment'>Apartment</option>
+                            <option value='Capsule Room'>Capsule Room</option>
+                            "), {}, style: "width: 330px", class: "roundedFormField" %>
```

Comment:
I just noticed. For the `select`, you can simplify it to the following:
`<%= f.select :home_type, {'Apartment': 'Apartment', 'House': 'House', 'Bed & Breakfast': 'Bed & Breakfast'}, { include_blank: 'Select...'}, { class: 'form-control w-100' , required: true } %>`

No need to use `option` element.